docs/getting-started/application/patterns.rst

```diff
@@ -282,303 +176,26 @@ We can combine field access control with our record level access control
     We use the ``oso.query`` method in this example to query a rule other than
     ``allow``.
 
-.. todo::
-
-    relevant link & this is incorrect with our API now!
-
-In this example, we evaluated both record & column level access control after
-data fetch.  However, it may be more efficient to use column level access
-control to only load the columns the user can access:
-
-.. code-block:: python
-    :caption: :fab:`python`
-
-    from oso import Variable
-
-    def get_expense(user, expense_id):
-        # Query oso for all fields allowed for this user.
-        allowed_fields = oso.query_rule("allow_field",
-                                   user, "view", expense, Variable("field"))
-        # Convert the returned query response into a list of fields
-        allowed_fields = [r["field"] for r in allowed_fields]
-        allowed_fields_sql = db.sql_escape(allowed_fields.join(", "))
-
-        expense = db.fetch(
-            f"SELECT {allowed_fields_sql} FROM expenses WHERE id = %",
-            expense_id)
-
-        # Record level authorization.
-        if oso.is_allowed(user, "view", expense):
-            # Return only authorized_data to the user.
-            ...
-        else:
-            # Not authorized
-            return NotAuthorizedResponse()
-
-Now, we are using oso to tell us what fields to query for.  In this example, the
-policy is evaluated both **before and after data fetch** for greater efficiency.
-
-.. admonition:: Variables provide flexibility
-
-    Notice that we didn't have to change our policy file at all to make this
-    change from the previous example. We passed in a ``Variable`` which
-    instructs oso to find all values of ``field`` that match the rules we
-    defined in our policy.  This flexibility derives directly from writing a
-    :doc:`declarative policy in Polar </more/language/polar-foundations>`!
-
-.. _authorizing-list-endpoints:
-
-Authorizing List Endpoints
```

Comment:
I think we should keep most of this with maybe some wording updates and more context about how this compares to data filtering

Reply:
We should probably split it into a separate page about data filtering